### PR TITLE
TouchSensor to FLU

### PR DESCRIPTION
**Pull request by mael25**
**Description**
Switch the `TouchSensor` (type=force) measurement axis form z to x, in order to meet FLU convention.

**Tasks**
- [x] Add illustration

**Documentation**
https://cyberbotics.com/doc/reference/touchsensor?version=feature-enu-touchsensor

### CODE REVIEW

**Comment by lukicdarkoo** [comment]
@cyberbotics/maintainers I would add an image of each sensor (not just specific to this PR) in the documentation that clearly depicts the axes. I started it here:
https://cyberbotics.com/doc/reference/lidar?version=feature-lidar-flu#description

That way we will establish clear conventions and it will be easier for the users to get familiar with the sensors. A few guidelines:
- The sensor image should contain a context, like a typical physical sensor, robot, and environment. Additional sketches (without context) can be included if necessary.
- The context (robot and environment) should be simple, it should not distract a user.
- The axes should be clearly visible.
- The important parts of the sensors should be tagged.
- If the optional rendering (`View > Optional Rendering`) is available for the sensor it should be visible in the picture (and tagged).

**Comment by mael25** [comment]
@lukicdarkoo I have added a picture (https://cyberbotics.com/doc/reference/touchsensor?version=feature-enu-touchsensor#coordinate-system) with the coordinate system of the `TouchSensor`, let me know if you think it is sufficient to understand the device.

File: docs/reference/touchsensor.md [hunk 1]
```diff
@@ -42,11 +42,11 @@ The [Physics](physics.md) node of a "bumper" sensor is not required.
...
-So the "force" sensor returns the projection of the force on its z-axis; a force perpendicular to the z-axis yields zero.
-For this reason, a "force" sensor must be oriented such that its positive z-axis points outside of the robot, in the direction where the force needs to me measured.
-For example if the [TouchSensor](#touchsensor) is used as foot sensor then the z-axis should be oriented downwards.
+A "force" [TouchSensor](#touchsensor) computes the (scalar) amount of force currently exerted on the sensor's body along the x-axis.
+The sensor uses this equation: *r=|f|*cos(α)*, where *r* is the return value, *f* is the cumulative force currently exerted on the sensor's body, and *α* is the angle between *f* and the sensor's x-axis.
+So the "force" sensor returns the projection of the force on its x-axis; a force perpendicular to the x-axis yields zero.
+For this reason, a "force" sensor must be oriented such that its positive x-axis points outside of the robot, in the direction where the force needs to me measured.
+For example, if the [TouchSensor](#touchsensor) is used as foot sensor, then the x-axis should be oriented downwards.
```
Comment (lukicdarkoo) [inline docs/reference/touchsensor.md:49]:
```suggestion
For example, if the [TouchSensor](#touchsensor) is used as a foot sensor then the x-axis should be oriented downwards.
```

**Review by lukicdarkoo — CHANGES_REQUESTED** [review]
I believe we can merge this one without affecting the other PRs

**Review by lukicdarkoo — APPROVED** [review]
Thank you

**Pull request merged**